modules/apps/portal-scheduler/portal-scheduler/src/main/java/com/liferay/portal/scheduler/internal/SchedulerEngineHelperImpl.java

```diff
@@ -84,7 +88,8 @@
 	configurationPid = "com.liferay.portal.scheduler.internal.configuration.SchedulerEngineHelperConfiguration",
 	enabled = false, service = SchedulerEngineHelper.class
 )
-public class SchedulerEngineHelperImpl implements SchedulerEngineHelper {
+public class SchedulerEngineHelperImpl
+	implements ConfigurationModelListener, SchedulerEngineHelper {
```

Comment:
@marianoalvarosaiz This `ConfigurationModelListener` seems not needed, as you register an anonymous one in activate.

Reply:
@dantewang You're right. My mistake. Not needed. I left it unintentionally from a previous version.